### DISCUSSION TITLE
Office hours recording

### DISCUSSION THREAD

**Discussion by oye-bobs** [cu-esiil-edu/2024-data-short-course#51] [General]
Hi @eculler I noticed that Wednesday office hours for 17th of April hasn't been added to the syllabus, I wasn't available during that time can I get the recording?

**Comment by nquarder** [comment]
hi @oye-bobs ! here is a link to the recording from yesterday's office hours:

https://cuboulder.zoom.us/rec/share/Xfd-8zt9n3LckulQJINc6SFfptcixgPKp1Z5nmIoqe7-XKpJnMh5J-GodGieVrgU.q7fqWjRRUwfmkiba

I will also add to the syllabus page. We went through a good demonstration w/ @gauravchakrabarty about how to add a map to your portfolio page. Also spent time troubleshooting w/ @chyasirabbasjutt but ran out of time - will circle back to this today hopefully if there is time.

**Reply by gauravchakrabarty** [reply]
@nquarder Thank you

**Comment by chyasirabbasjutt** [comment]
Thank you sir.
